clang/lib/CIR/CodeGen/CIRGenBuiltinAArch64.cpp

```diff
@@ -2197,6 +2197,14 @@ mlir::Value buildNeonCall(unsigned int builtinID, CIRGenFunction &cgf,
   }
 }
 
+/// Get integer from a mlir::Value that is an int constant or a constant op.
+static int getIntFromMLIRValue(mlir::Value val) {
+  auto constOp = mlir::cast<mlir::cir::ConstantOp>(val.getDefiningOp());
```

Comment:
Out of curiosity, I thought I had already commented on this, was this part of another PR?
- `getIntFromMLIRValue` -> `getIntValueFromConstOp`.
- This should return `int64_t`, if you wanna cast down the result that should be done in the caller.

Reply:
No, this was only introduced in this PR. Sure. I'll change this helper function on you advices. 
Q: should we move it to CIRGenBuilder.h to make it a common utility?

Reply:
Maybe if it gets more uses we can do it that, sounds good for now!